spacepy/pycdf/const.py

```diff
@@ -262,7 +268,15 @@
 IS_A_NETCDF =  -2228
 TT2000_TIME_ERROR = -2229
 UNABLE_TO_PROCESS_CDF = -2230
-ZLIB_DECOMPRESSION_ERROR = - 2231
+ZLIB_COMPRESS_ERROR = -2231
```

Comment:
`ZLIB_DECOMPRESSION_ERROR` has been removed, but had the same value that's now assigned to `ZLIB_COMPRESS_ERROR`. What version range of CDF is the new value valid for, and could this cause any misreporting in anything we support?

Reply:
Ultimately we don't parse the error codes, they get passed to the CDF library's parser. At some point they changed from ZLIB_DECOMPRESSION_ERROR to two codes, ZLIB_COMPRESS_ERROR and ZLIB_UNCOMPRESS_ERROR. This change is present in 3.5, which is our oldest supported version.

We don't use this value for anything. I suppose we could include both names (both set to -2231) in case someone is using the old one, but given our CDF version support, I feel safe just using the current name.

Reply:
I'm happy to have this as-is if it's from 3.5 since that's a decade old at this point.